client/my-sites/themes/collections/collection-definitions.tsx

```diff
@@ -85,88 +30,8 @@ export const THEME_COLLECTIONS = {
 		fullTitle: translate( 'Partner Themes' ),
```

Comment:
GH doesn't let me comment the correct line, but anyway.
Let's update the short title to `Partner Themes` as well, since it's not repetitive anymore.

```diff
-		title: translate( 'Partner' ),
+		title: translate( 'Partner Themes' ),
```

Reply:
Done.